src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -225,7 +225,7 @@ public MSBuildClientExitResult Execute(CancellationToken cancellationToken)
 
                 ReadPacketsLoop(cancellationToken);
 
-                MSBuildEventSource.Log.MSBuildServerBuildStop(descriptiveCommandLine, _numConsoleWritePackets, _sizeOfConsoleWritePackets, _exitResult.MSBuildClientExitType.ToString(), _exitResult.MSBuildAppExitTypeString);
```

Comment:
Is string.Empty the right default here? I'm wondering if we should put "exit type could not be determined" or something, though ideally we should always know the exit type.

Reply:
Agreed that we should always know it. I don't feel too bad about empty-string here because we're also logging the enum value.